libs/domain/feature-loaders/src/snapshot/SnapshotProposalLoader.ts

```diff
@@ -0,0 +1,187 @@
+import { createGraphQLClient, GraphQLClient, ProgramError } from "@banklessdao/util-data";
+import { InitContext, InternalLoadContext, LoadContext, ScheduleMode } from "@shared/util-loaders";
+import { Data, Nested, NonEmptyProperty, OptionalNumberArrayOf, OptionalObjectRef, OptionalProperty, OptionalStringArrayOf, Property, RequiredArrayRef, RequiredStringArrayOf } from "@banklessdao/util-schema";
+import { DocumentNode } from "graphql";
+import gql from "graphql-tag";
+import * as t from "io-ts";
+import * as TE from "fp-ts/TaskEither";
+import { GraphQLDataLoaderBase } from "../base/GraphQLDataLoaderBase";
+import { BATCH_SIZE } from "../defaults";
+import { withMessage } from "io-ts-types";
+
+export const URL = "https://hub.snapshot.org/graphql";
+
+export const makeQUERY: (space:String)=>DocumentNode = (space) => {
```

Comment:
In Typescript is is recommended to use the built-in types. In this case `String` -> `string`. I've also noticed that you're not using a code formatter. I'd recommend installing the *Prettier* plugin to VS Code. It should work automatically (there is a config in `.prettierrc` for it. Using a formatter is important to prevent having merge conflicts because of mixed formatting rules.

Reply:
NIT: you can shorten
```ts
export const makeQUERY: (space:String)=>DocumentNode = (space) => {
```
to 
```ts
export const makeQUERY = (space: string): DocumentNode => {
```
It also helps prevent some bugs too.

Reply:
- [X] `String -> string` 
- [X] Code formatter
- [X] Shorter function definition